### PR TITLE
Update azure-pipelines.yml

### PR DESCRIPTION
**Pull request by behoyh**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -65,7 +65,14 @@ steps:
...
+- task: AzureWebApp@1
+  displayName: 'Deploy Azure App Service'
   inputs:
-    PathtoPublish: '$(Build.ArtifactStagingDirectory)\ArtifactsToBePublished'
+    azureSubscription: 'hazzat-api-ppe - Azure'
+    appType: 'webApp'
+    appName: 'hazzat-api-ppe'
+    deployToSlotOrASE: true
+    resourceGroupName: 'hazzat-api-ppe-rg'
+    slotName: 'production'
```
Comment (fadyham) [inline azure-pipelines.yml]: I don't have slots in the ppe environment.  It's not available in this sku.  Will this work still?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -65,7 +65,14 @@ steps:
     Contents: '$(Build.BuildId).zip'
     TargetFolder: '$(Build.ArtifactStagingDirectory)\ArtifactsToBePublished'
 
-- task: PublishBuildArtifacts@1
-  displayName: 'Publish Artifact: drop'
+- task: AzureWebApp@1
```
Comment (fadyham) [inline azure-pipelines.yml]: Does this overwrite existing deployment somehow?

File: azure-pipelines.yml (hunk 1 above)
Comment (behoyh) [reply azure-pipelines.yml]: [Yes](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-rm-web-app-deployment?view=azure-devops), the flag `deployToSlotOrASE` works for either or as long as the `slotName` is set to `production`

File: azure-pipelines.yml (hunk 2 above)
Comment (behoyh) [reply azure-pipelines.yml]: From what I understand, MSDeploy removes non-matching files at the destination [non-optionally](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-rm-web-app-deployment?view=azure-devops#web-deploy).

> # Remove additional files at destination: 
> Deletes files in the Azure App Service that have no matching files in the App Service artifact package or folder being deployed.

**Pull request merged**